cluster-up/cluster/kind-k8s-1.17/provider.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+export CLUSTER_NAME="kind-1.17"
+export KIND_NODE_IMAGE="kindest/node:v1.17.2"
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function up() {
+    cp $KIND_MANIFESTS_DIR/kind.yaml ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml
```

Comment:
In the IPv6 provider.sh we do way more: https://github.com/kubevirt/kubevirtci/blob/ec741ec3a2ad418457c81be260ddf5bc1941f3ca/cluster-up/cluster/kind-k8s-1.17.0-ipv6/provider.sh

I get that this is all we need, but I hate seeing the providers getting out of sync and being implicitly specific to their use-cases.

Would you abstract the IPv6 setup regarding storage and others and use it in both providers?

Reply:
ah thanks for the explanation , forgot that

hmm about the abstract
i am not sure this is what Roman strived to do ?
i think he separated it by meaning ?
i had a conversion with him on that, maybe worth asking so it wont 
intersect with his new design ?
i might be mistaken here, just recalled for sure we did speak about 
making providers core separated even that it mean less code reuse.

Reply:
@phoracek as the @rmohr effort of untangling providers (meaning we don't re-use we have all at one place) I will copy the storage code at the provider.

Reply:
Ack